CONTRIBUTING.md

```diff
@@ -3,13 +3,29 @@ Contributing
 
 It's good to hear that you want to contribute to Pony!
 
-First of all please [search existing issues][complete-issue-list]. Apart from bug reports you will find feature requests and work currently in progress too. If you cannot find a suitable issue — [create a new one][new-issue]. Potential pull requests are no exception. Before you start working, let's have a discussion and agree on the solution and scope.
+There are a number of ways to contribute to Pony. As this document is a little long, feel free to jump to the section that applies to you currently:
 
-Each of [bug report](#bug-report), [feature request](#feature-request) or [pull request](#pull-request)
-has to cover different aspects described below.
+* [Feature request](#feature-request)
+* [Bug report](#bug-report)
+* [Pull request](#pull-request)
+
+Additional notes regarding formatting:
+
+* [Documentation formatting](#documentation-formatting)
+* [Code formatting](#code-formatting)
+
+Feature request
+---------------
+To request a change to the way that Pony works, please open an issue in the [RFCs repository](https://github.com/ponylang/rfcs) rather than this one. New features and other significant language changes must go through the RFC process.
```

Comment:
I don't want to duplicate the enumeration of when things apply from the RFC page as that leads to having to keep them in sync, the idea was just enough info to get people to go read the info in the RFC repo. Thoughts on how to make it more "GO READ THAT".

It could be.

"STOP! For any feature requests or enhancements to the Pony distribution, it is quite likely that you have to go through our RFC process. Before opening or submitting any feature requests, please make sure you are familiar with the RFC process and follow the process as required."